### ISSUE TITLE
Prominent search feature

### ISSUE THREAD

**Issue by matatk** [ThePacielloGroup/glossary#2]
A.k.a. a big search box right at the start on the front page, where people can type in something and, as they type, the results will home in on the term/acronym for which they're looking, like http://caniuse.com

**Comment by stevefaulkner** [comment]
search script?

**Comment by patrickhlauke** [comment]
type-ahead search? search suggestions?

**Comment by matatk** [comment]
Yes; I think the way it works on CanIUse is ideal (though naturally we can tweak if need be).  Would need to do this on the client side if hosting via GitHub, so would need some sort of JSON thing to be built as part of the build process, but it's all do-able.
